base/src/commonMain/kotlin/monitor/ServiceMonitor.kt

```diff
@@ -15,7 +15,7 @@
 
  */
 
-package com.splendo.kaluga.base
+package com.splendo.kaluga.base.monitor
```

Comment:
FYI @corrado4eyes also interested what you think of the package change.. I guess this does not break too much (as what you end up using is in `system` etc) but it might break your PR